src/property-filter/index.tsx

```diff
@@ -42,12 +52,18 @@ const PropertyFilter = React.forwardRef(
       },
     };
 
+    if (hideOperations && enableTokenGroups) {
+      warnOnce('PropertyFilter', 'Operations cannot be hidden when token groups are enabled.');
+      hideOperations = false;
```

Comment:
Is this the correct fallback? If the operations are hidden it's presumably because the API does not support it whereas isn't token groups just a visual treatment?